packages/assets/src/tools/validate-files.ts

```diff
@@ -30,14 +58,27 @@ async function validateValidOrgs() {
     const diff = difference(expectedFiles.files, orgAssets.files);
 
     if (diff.length > 0) {
-      hasErrors = true;
-      console.error(
-        `
-${themeVariantAsString(orgAssets.org)} is missing some assets:
-${diff.join('\n')}
-
-`,
+      console.warn(
```

Comment:
Not sure if this needs a warning if it is expected behaviour?

Reply:
Well, yes, it will become expected behaviour for some files, but I guess having it as a normal console.log will be enough for notifying developers that they may have forgotten to add a file.